File: 12.1/money.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

long long solution(int price, int money, int count) {
    int i =0;
    long long sum=0;
    while(++i<= count)
         sum += (price * i);
    return  sum > money ? sum- money :0;
}

File: 12.1/money.py
def solution(price, money, count):
    sum =0
    for i in range(1,count+1): // 파이썬 for문  1 부터 cnt -1 까지돈다 
        sum += price * i
    if sum >= money:
        answer = sum - money
    else :
        answer =0
        
    return answer

